src/url-utils.js

```diff
@@ -53,9 +53,9 @@ export function repoBaseUrl(repo, gateway = ipfsDefaultConf.gateway) {
   )
 }
 
-// Removes the HTTP protocol of a URL
+// Removes the HTTP protocol of a URL, and the final slash.
 export function stripUrlProtocol(url = '') {
```

Comment:
Maybe `stripUrlProtocolAndFinalSlash`?

Reply:
Yes… at some point I called it `displayUrl` or something like that, maybe it would be better as it is the intention really. Maybe `formatUrlForDisplay()` would work better. I’m going to wait for the PRs relying on this branch to be merged, then I’ll change it once everything is merged in.